src/plugins/mvcc_delete_plugin.cpp

```diff
@@ -0,0 +1,177 @@
+#include "mvcc_delete_plugin.hpp"
+
+#include "operators/get_table.hpp"
+#include "operators/table_wrapper.hpp"
+#include "operators/update.hpp"
+#include "operators/validate.hpp"
+#include "storage/pos_list.hpp"
+#include "storage/reference_segment.hpp"
+#include "storage/table.hpp"
+
+namespace opossum {
+
+MvccDeletePlugin::MvccDeletePlugin()
+    : _sm(StorageManager::get()),
+      _delete_threshold_rate_invalidated_rows(0.8),
```

Comment:
Have you thought about using constants here so the parameter can be better configured?